lib/screens/util/last_deploy/s3_fetch.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Screens.Util.LastDeploy.S3Fetch do
+  @moduledoc false
+
+  @bucket "mbta-screens"
+
+  def get_last_deploy_time do
+    env = Application.get_env(:screens, :environment_name, "screens-prod")
```

Comment:
If environment name is not found, it defaults to prod. I noticed elsewhere, we vary in whether we default to dev or prod. Why did you choose prod here?

Reply:
I can't speak for the other places, but here I don't really have a good reason for defaulting to prod. In a couple of the other places, it is dealing with where assets are stored in S3. That seems fine because that will allow us to see whatever assets would show on live screens by default. The same argument could be made here: if you can't figure out what ENV you're in, just look at the time prod last deployed and use that.